app/src/components/Statistics/Statistics.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { StatisticsElement } from "./StatisticsElement";
+import brandIcon from "../../images/icon-brand-recognition.svg";
```

Comment:
I think that those icons should be uppercase

Reply:
corrected.